tests/unit/amo/pages/TestAddonReviewList.js

```diff
@@ -114,14 +114,68 @@ describe(__filename, () => {
   };
 
   describe('<AddonReviewList/>', () => {
-    it('shows placeholders for reviews without an addon', () => {
```

Comment:
ah, I missed this test case. Why did you remove it actually?

Reply:
Whoops - didn't mean to